src/home/EigenTVLOvertimeChart.jsx

```diff
@@ -0,0 +1,232 @@
+          rx={14}
+        />
+        <Group left={margin.left} top={margin.top}>
+          <GridRows
+            scale={tvlScale}
+            width={innerWidth}
+            height={innerHeight}
+            stroke="#7A86A5"
+            strokeOpacity={0.2}
+            numTicks={getNumberOfTicks(width, 'y')}
+          />
+          <GridColumns
+            scale={dateScale}
+            width={innerWidth}
+            height={innerHeight}
+            stroke="#7A86A5"
+            strokeOpacity={0.2}
+            numTicks={getNumberOfTicks(width, 'x')}
+          />
+
+          <AxisLeft
+            scale={tvlScale}
+            left={0}
+            tickFormat={formatNumber}
+            tickLabelProps={() => ({
+              className: 'fill-foreground-active text-xs',
+              fontSize: 11,
+              textAnchor: 'end'
+            })}
+          />
+
+          <AxisBottom
+            scale={dateScale}
+            tickFormat={date => formatDateToVerboseString(date)}
+            top={innerHeight}
+            tickLabelProps={() => ({
+              className: 'fill-foreground-active text-xs',
+              textAnchor: 'middle'
+            })}
+            tickValues={data
+              .filter(
+                (_, i) =>
+                  i %
+                    Math.max(
+                      1,
+                      Math.floor(data.length / getNumberOfTicks(width, 'x'))
+                    ) ===
+                  0
+              )
+              .map(d => new Date(d.timestamp))}
+          />
+
+          <LinePath
+            className="stroke-purple-500"
+            strokeWidth={2}
+            data={data}
+            x={d => dateScale(new Date(d.timestamp))}
+            y={d => tvlScale(getEthTVL(d))}
+          />
+
+          <LinePath
+            className="stroke-purple-300"
+            strokeWidth={2}
+            data={data}
+            x={d => dateScale(new Date(d.timestamp))}
+            y={d => tvlScale(getLstTVL(d))}
+          />
+
+          {tooltipData && (
+            <>
+              <Circle
+                cx={dateScale(new Date(tooltipData.timestamp))}
+                cy={tvlScale(getEthTVL(tooltipData))}
+                r={4}
+                className="fill-purple-500"
+                stroke="white"
+                strokeWidth={2}
+              />
+              <Circle
+                cx={dateScale(new Date(tooltipData.timestamp))}
+                cy={tvlScale(getLstTVL(tooltipData))}
+                r={4}
+                className="fill-purple-300"
+                stroke="white"
+                strokeWidth={2}
+              />
+            </>
+          )}
+          <rect
+            x={0}
+            y={0}
+            width={innerWidth}
+            height={innerHeight}
+            fill="transparent"
+            onTouchStart={handleTooltip}
+            onTouchMove={handleTooltip}
+            onMouseMove={handleTooltip}
```

Comment:
hm why do we need so many events here?
```suggestion
            onTouchStart={handleTooltip}
            onTouchMove={handleTooltip}
            onMouseMove={handleTooltip}
```

Reply:
yes these are for touch screen devices